src/PolySharp.SourceGenerators/EmbeddedResources/System.Diagnostics.UnreachableException.cs

```diff
@@ -0,0 +1,42 @@
+// <auto-generated/>
+#pragma warning disable
+#nullable enable annotations
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics
+{
+    /// <summary>
+    /// Exception thrown when the program executes an instruction that was thought to be unreachable.
+    /// </summary>
+    public sealed class UnreachableException : Exception
```

Comment:
Should this be `internal` by default? See #92 for a similar case.

Reply:
Rebased and addressed in a2c4dd2edb0d8a6053080d78902558a7c20e3243